modules/sandboxed-containers-selecting-nodes.adoc

```diff
@@ -3,5 +3,68 @@
 // * sandboxed_containers/deploying_sandboxed_containers.adoc
 
 [id="sandboxed-containers-selecting-nodes_{context}"]
+= Selecting nodes for {sandboxed-containers-first}
 
-= Selecting nodes for OpenShift sandboxed containers
+You can selectively install the Kata runtime on specific workers.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have installed the OpenShift CLI (oc).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Identify the labels that you want to use for selecting your nodes. For this example, select nodes with labels to be chosen as candidates to run on your {sandboxed-containers-first} workloads. If the nodes exist, they are selected.
+.. To apply a label to a node, run the following command:
++
+[source,terminal]
+----
+$ oc label node <worker_node_name> <label>=<value>
+----
++
+This labels your worker node with the `<label>` label that has a value of `<value>`.
+
+. To add a label selector, edit the `KataConfig` custom resource (CR):
++
+[source,terminal]
+----
+$ oc edit kataconfig
+----
+.Example
++
+[source,yaml]
+----
+  apiVersion: kataconfiguration.openshift.io/v1
+  kind: KataConfig
+  metadata:
+    name: cluster-kataconfig
+  spec:
+    kataConfigPoolSelector:
+      matchLabels:
+         custom-kata-machine-pool: true
+----
+. Use `oc apply -f <kata-config-with-label-selector.yaml>` to apply the label to a node.
```

Comment:
If you use `oc edit kataconfig`, there is no need to `oc apply -f` after that, is there?

Reply:
yes, that's true

Reply:
@c3d @jensfr - what should this change to?

Reply:
I was going to point out the same thing.
I think you can remove the step. Just editing the object is sufficient to add the label selector. @c3d @jensfr  Please correct me if I'm wrong.

Reply:
yes, you can remove the 'oc apply -f ...' step